tempto-kafka/src/main/java/io/prestodb/tempto/fulfillment/table/kafka/KafkaMessageContentsBuilder.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.prestodb.tempto.fulfillment.table.kafka;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+import static parquet.Preconditions.checkArgument;
+
+public class KafkaMessageContentsBuilder
+{
+    private final ByteArrayOutputStream contents = new ByteArrayOutputStream();
+
+    public static KafkaMessageContentsBuilder contentsBuilder() {
+        return new KafkaMessageContentsBuilder();
+    }
+
+    public KafkaMessageContentsBuilder appendBytes(byte[] bytes) {
+        try {
+            contents.write(bytes);
+        }
+        catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        return this;
+    }
+
+    public KafkaMessageContentsBuilder appendBytes(int ... bytes) {
+        for (int b : bytes) {
+            checkArgument(b >= 0 && b <= 255, "Byte must be in 0-255 range");
+            contents.write(b);
+        }
+        return this;
+    }
+
+    public KafkaMessageContentsBuilder appendIntLittleEndian(int v) {
+        contents.write((v & 0xff000000) >> 24);
+        contents.write((v & 0xff0000) >> 16);
+        contents.write((v & 0xff00) >> 8);
+        contents.write(v & 0xff);
```

Comment:
I lied to you, https://en.wikipedia.org/wiki/Endianness#Illustration

Reply:
That is true :)
The method does what I want though. I will change the name :)